package.xml

```diff
@@ -1,18 +1,23 @@
-<package>
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>geometric_shapes</name>
-  <version>0.6.1</version>
+  <version>0.6.2</version>
   <description>This package contains generic definitions of geometric shapes and bodies.</description>
 
   <author email="isucan@google.com">Ioan Sucan</author>
   <author email="gjones@willowgarage.edu">Gil Jones</author>
 
   <maintainer email="dave@dav.ee">Dave Coleman</maintainer>
   <maintainer email="130s@2000.jukuin.keio.ac.jp">Isaac I. Y. Saito</maintainer>
+  <maintainer email="vmayoral@acutronicrobotics.com">Víctor Mayoral Vilches</maintainer>
```

Comment:
Does Victor really become a maintainer of this package?

Reply:
I don't think so. By now this is just a fixup of the existing branch based on Acutronic's fork. I'll remove all unnecessary changes when cleaning up the history